frontend/apps/reader/modules/readergesture.lua

```diff
@@ -118,6 +123,9 @@ local default_multiswipes = {
     "northeast southeast",
     -- "southwest northwest", -- visually ambiguous
     -- "northwest southwest", -- visually ambiguous
+    "northwest southwest northwest",
```

Comment:
That one is as ambiguous as the previous one I didn't dare added :) So, we might as well dare.

Reply:
Mhhh, actually it is not :) it reads the same backwards following the arrows :)